### PR TITLE
Little review on the current documentation

### PR DESCRIPTION
**Pull request by pakb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Translations can then be updated with
 ```bash
 summon -p gopass npm run update:translations
 ```
+The file `secrets.yml` will tell `gopass` which keys to get from the store.
```
Comment (ltshb) [inline README.md:99]: Actually `secrets.yml` is used by summon not gopass. It does the mapping between the gopass key and the environment variable that should be exported

**Review by ltshb — CHANGES_REQUESTED** [review]
Looks good just a small changes about `secrets.yml` and some suggestions.

For new comer (like me) it would be good to have a link to `Vue` and `Vuex` in the `Project structure`  chapter.

In the `Project structure` a Folder tree image of the structure would help to understand it.

In chapter `Tooling for translation update` I would specify the gopass store name: `infra-gopass-bgdi`

**Comment by pakb** [comment]
Should be good, I've added what you suggested, thanks!

**Review by ltshb — APPROVED** [review]
:+1:

**Pull request merged**